sdk/eventhub/event-hubs/src/impl/patitionKeyToIdMapper.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint-disable no-fallthrough */
+
+import os from "os";
```

Comment:
os might not work in browser?

Reply:
Yeah many things here are node specific such as `os` and `Buffer` but the tests passed so I assumed there is polyfilling going on. I will revisit the browser support.

Reply:
rollup shims perhaps